core/src/main/java/com/github/mizool/core/converter/EnumConverter.java

```diff
@@ -20,11 +20,22 @@
 {
     public String fromPojo(Enum<?> enumValue)
     {
-        return enumValue.name();
+        String result = null;
+
+        if (enumValue != null)
+        {
+            result = enumValue.name();
+        }
+        return result;
     }
 
     public <T extends Enum<T>> T toPojo(Class<T> enumClass, String value)
     {
-        return T.valueOf(enumClass, value);
+        T result = null;
+        if (enumClass != null && value != null)
```

Comment:
`value` can indeed be null (e.g. in an optional field), that part is correct.

however, if `enumClass` is null, that is a programming error we should not swallow silently. remove that condition, let's run into the NPE thrown by `Enum.valueOf()` in that case.